### DISCUSSION TITLE
Chinese patch

### DISCUSSION THREAD

**Discussion by MyGox** [maruohon/tweakeroo#373] [Ideas]
I hope to launch a Chinese version

**Comment by maruohon** [comment]
What do you mean exactly? Adding a Chinese localization file?

**Reply by myunco** [reply]
Yes, do you have a plan for this? I wish add a Chinese localization file.

**Reply by Poopooracoocoo** [reply]
isn't that https://github.com/maruohon/tweakeroo/pull/118?
